mlos_core/mlos_core/version.py

```diff
@@ -7,4 +7,7 @@
 """
 
 # NOTE: This should be managed by bumpversion.
-_VERSION = '0.5.1'
+VERSION = '0.5.1'
```

Comment:
Make some variables non-private.  Was unnecessary and removes some pylint exceptions.